### PR TITLE
add sql statement about pump and drainer

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? (Required)

 adding sql statement about pump and drainer

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: sql-statements/sql-statement-change-drainer.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (bb7133) [inline sql-statements/sql-statement-change-drainer.md:1]: Please add those new syntaxes to `/TOC.md`

Comment (WangXiangUSTC) [reply sql-statements/sql-statement-change-drainer.md:1]: 已经更新

**Comment by bb7133** [comment]
Please update the `TOC.md` @WangXiangUSTC , thanks~

**Comment by WangXiangUSTC** [comment]
> Please update the `TOC.md` @WangXiangUSTC , thanks~

@bb7133 done

File: sql-statements/sql-statement-change-drainer.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+category: reference
+---
+
+# CHANGE DRAINER
+
+`CHANGE DRAINER` 语句用于修改集群中 Drainer 的状态信息。
+
+> **注意：**
+>
+> Drainer 在正常运行时会自动上报状态到 PD，仅在 Drainer 处于异常情况时实际状态与 PD 中保存的状态信息不一致时，使用该语句修改 PD 中存储的 Drainer 状态信息。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-drainer.md:13]:
```suggestion
> Drainer 在正常运行时会自动上报状态到 PD，仅在 Drainer 处于异常情况时或实际状态与 PD 中保存的状态信息不一致时，使用该语句修改 PD 中存储的 Drainer 状态信息。
```

Comment (toutdesuite) [reply sql-statements/sql-statement-change-drainer.md:13]: 请确认修改后的句子是否意思正确~

File: sql-statements/sql-statement-change-pump.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+title: CHANGE PUMP
+summary: TiDB 数据库中 CHANGE PUMP 的使用概况。
+category: reference
+---
+
+# CHANGE DRAINER
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-pump.md:7]:
```suggestion
# CHANGE PUMP
```

File: sql-statements/sql-statement-change-pump.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+`CHANGE PUMP` 语句用于修改集群中 Pump 的状态信息。
+
+> **注意：**
+>
+> Pump 在正常运行时会自动上报状态到 PD，仅在 Pump 处于异常情况时实际状态与 PD 中保存的状态信息不一致时，使用该语句修改 PD 中存储的 Pump 状态信息。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-pump.md:13]: ditto

File: sql-statements/sql-statement-change-drainer.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+可以看出 `drainer1` 已经超过一天没有更新状态，该 Drainer 处于异常状态，但是 State 仍然为 `Online`，使用 `CHANGE DRAINER` 将该 Drainer 状态修改为 `paused`：
+
+{{< copyable "sql" >}}
+
+```sql
+CHANGE DRAINER TO NODE_STATE ='paused' FOR NODE_ID 'drainer1';
+```
+
+```
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-drainer.md:42]:
```suggestion
```sql
```

File: sql-statements/sql-statement-change-pump.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+## 示例
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW PUMP STATUS;
+```
+
+```
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-pump.md:23]:
```suggestion
```sql
```

Comment (toutdesuite) [reply sql-statements/sql-statement-change-pump.md:23]: sql 语句的执行结果也建议加上"sql"，下同

File: sql-statements/sql-statement-show-pump-status.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+title: SHOW PUMP STATUS
+summary: TiDB 数据库中 SHOW PUMP STATUS 的使用概况。
+category: reference
+---
+
+# SHOW DRAINER STATUS
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-pump-status.md:7]:
```suggestion
# SHOW PUMP STATUS
```

**Comment by toutdesuite** [comment]
@WangXiangUSTC 确认一下这个适用于 master 分支吗？
image

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC 确认一下这个适用于 master 分支吗？
> image

@bb7133 please check it

File: sql-statements/sql-statement-change-drainer.md (hunk 2 above)
Comment (toutdesuite) [reply sql-statements/sql-statement-change-drainer.md:13]: 好的，已更新~

File: sql-statements/sql-statement-change-pump.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+category: reference
+---
+
+# CHANGE PUMP
+
+`CHANGE PUMP` 语句用于修改集群中 Pump 的状态信息。
+
+> **注意：**
+>
+> Pump 在正常运行时会自动上报状态到 PD，仅在 Pump 处于异常情况时实际状态与 PD 中保存的状态信息不一致时，使用该语句修改 PD 中存储的 Pump 状态信息。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-change-pump.md:13]:
```suggestion
> Pump 在正常运行时会自动上报状态到 PD，仅在 Pump 处于异常情况导致实际状态与 PD 中保存的状态信息不一致时，使用该语句修改 PD 中存储的 Pump 状态信息。
```

File: sql-statements/sql-statement-show-master-status.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
+---
+title: SHOW MASTER STATUS
+summary: TiDB 数据库中 SHOW MASTER STATUS 的使用概况。
+category: reference
+---
+
+# SHOW MASTER STATUS
+
+`SHOW MASTER STATUS` 语句用于显示集群当前最新的 tso 信息。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-master-status.md:9]:
```suggestion
`SHOW MASTER STATUS` 语句用于显示集群当前最新的 TSO 信息。
```

File: sql-statements/sql-statement-show-master-status.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+| File        | Position           | Binlog_Do_DB | Binlog_Ignore_DB | Executed_Gtid_Set |
++-------------+--------------------+--------------+------------------+-------------------+
+| tidb-binlog | 416916363252072450 |              |                  |                   |
++-------------+--------------------+--------------+------------------+-------------------+
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+`SHOW MASTER STATUS` 语句与 MySQL 兼容，但是执行结果有差异，在 MySQL 中执行结果为 binlog 的位置信息，而在 TiDB 中为最新的 tso 信息。
```
Comment (toutdesuite) [inline sql-statements/sql-statement-show-master-status.md:30]:
```suggestion
`SHOW MASTER STATUS` 语句与 MySQL 兼容，但是执行结果有差异，在 MySQL 中执行结果为 binlog 的位置信息，而在 TiDB 中为最新的 TSO 信息。
```

**Comment by bb7133** [comment]
LGTM

**Pull request merged**